src/main/java/com/google/devtools/build/lib/sandbox/AbstractSandboxSpawnRunner.java

```diff
@@ -125,16 +125,22 @@ private final SpawnResult run(
         sandbox.getSandboxExecRoot().getPathFile());
       String failureMessage;
       if (sandboxOptions.sandboxDebug) {
-        failureMessage =
-            CommandFailureUtils.describeCommandFailure(
-                true, sandbox.getArguments(), sandbox.getEnvironment(), execRoot.getPathString());
+      failureMessage =
```

Comment:
nit: Bogus indentation, here and below.

Reply:
The formatter was confused because this is correct but previous commits left the indent wrong on the previous lines. Fixed it.